arrays/arrays.c

```diff
@@ -29,7 +29,7 @@ Array *create_array(int capacity)
     newArr->count = 0;
 
     // Allocate memory for elements
-    newArr->elements = malloc(capacity * sizeof(char));
+    newArr->elements = malloc(capacity * sizeof(newArr->elements));
```

Comment:
this works because `newArr->elements` is a pointer (itself pointing to an array of (char *) ), so `sizeof()` still returns a pointer size. however, for humans reading this, `sizeof(char *)` better explains what you're trying to do

Reply:
lol, quoting me at myself